### DISCUSSION TITLE
Any Indian payment gateway

### DISCUSSION THREAD

**Discussion by daadu** [saleor/apps#1151] [Integrations & features]
## Some candidates
- RazorPay - https://razorpay.com/docs/api/payments/
- Instamojo - https://docs.instamojo.com/reference/payments-api

Feel free to suggest others in the comments below.

**Comment by daadu** [comment]
I saw https://github.com/saleor/saleor/pull/3205 merged, but no reference in the App Directory - https://docs.saleor.io/docs/3.x/developer/extending/apps/overview

**Reply by lkostrowski** [reply]
Hey, Razorpay is a plugin, you can find it Dashboard → Configuration → Plugins.

The page you mentioned lists apps, which are going to replace plugins in the future.
